app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -59,7 +59,7 @@ private void nextActivity() {
             LOGGER.info("Sending my profile with name: " + myProfile.getName() + ", public key: "
                     + myProfile.getPublicKey().toString());
             final Contact myProfileAsContact = new Contact(myProfile.getName(), new SimpleKey("simplekey", "73890ien"));
-            this.serviceLocator.getBluetoothHandler().send(myProfileAsContact);
+            this.serviceLocator.getBluetoothHandler().send(this.serviceLocator.getBluetoothHandler().objectToBytes(myProfileAsContact));
```

Comment:
> In a perfect world every serializable object knows how to serialize itself
There is a reason why I want that every class has a readObject and a writeObject or a SerializationProxy...

The reason why I promoted the method objectToBytes is because
- we do not want duplicate code, although both NFC and Bluetooth have to convert a message to bytes
- we want a function to have a single task, and send(Serializable) both converts the serializable object to bytes and makes sure that these bytes are sent
- I hate it as much as you do, but it seems that it's not possible to the NFCHandler send messages. It has to be done by the current activity. You cannot "send" messages, but you fill a buffer when the activity starts with the data you want to beam to the other activity. Android beams the buffer constantly to the ether. The send() method is therefore completely useless for NFC.